feg/gateway/tools/s8_cli/main.go

```diff
@@ -521,5 +521,8 @@ func main() {
 	}
 	args := os.Args[2:]
 	cmd.Flags().Parse(args)
-	os.Exit(cmd.Handle(args))
+	// "If a test calls os.Exit, the test binary will exit, and go test will report a failure." https://github.com/golang/go/issues/45619
+	if i := cmd.Handle(args); i != 0 {
+		os.Exit(i)
+	}
```

Comment:
maybe it would be best just remove the os.Exit(0) we have in this cli and just use return
And then just run the command in this line
```
cmd.Handle(args)
```

Reply:
See https://github.com/magma/magma/pull/12151/commits/321cf30f4591665c175a058e4c74ce96567a24ef